site/reference/pkg/init/README.md

```diff
@@ -3,31 +3,43 @@ title: "Init"
 linkTitle: "init"
 type: docs
 description: >
-   Initialize an empty package
+   Initialize an empty package.
 ---
 <!--mdtogo:Short
-    Initialize an empty package
+    Initialize an empty package.
 -->
 
-{{< asciinema key="pkg-init" rows="10" preload="1" >}}
+`init` initializes an existing empty directory as an empty kpt package.
```

Comment:
This currently creates a Kptfile and a README:
1. Should mention this in the reference docs.
2. The README needs to be updated as well.

Reply:
Update the description of the command to mention that the command will create a Kptfile and a README.md file.
Fixed the README to include the correct links.